### PR TITLE
added parsedown extension for header ids (#388)

### PR DESCRIPTION
**Pull request by lduer**
## Description

Contribution to #388 

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [x] My code follows the code style
- [x] All files have a license header 
- [x] All methods have a doc header with type declarations
- [ ] I have updated the documentation accordingly
- [x] I have added tests to cover my changes

### CODE REVIEW

**Comment by lduer** [comment]
As small addition, I've added some jquery functions to show/display the "link" icon when hovering the header (like in Github):

File: assets/sass/app.scss [hunk 1]
```diff
@@ -32,6 +32,22 @@ footer.main-footer {
     font-size: 80%;
 }
 
+/* ================================ help ================================ */
```
Comment (kevinpapst) [inline assets/sass/app.scss]: Can you create a new sass file for the documentation, as I like to improve the styles a little bit.
I think readability could be improved by increasing line height and some other font settings... but not in this PR, just so we have the sass file already in place.

**Comment by lduer** [comment]
The changes you requested are done. Is this PR merge-ready?

**Comment by kevinpapst** [comment]
Just one minor thing: I still have that styling issue I mentioned above in Safari and Chrome, both in mobile and full view.
I know there is not much space, tested with adding more padding to markdown-body but that is stealing a lot of space in the mobile view (fun fact: its also failing in the GitHub mobile view).
Mobile view be ignored, as there is no hover - but do you have an idea how to make Desktop look a bit less "disturbing".

**Comment by lduer** [comment]
Thank's for your additions and the merge.

**Comment by kevinpapst** [comment]
Thanks for your contribution, very welcome :smile:

**Pull request merged**